frontend/src/pages/ErrorsV2/ErrorStateSelect/ErrorStateSelect.tsx

```diff
@@ -288,7 +288,13 @@ const ErrorStateSelectImpl: React.FC<Props> = ({
 										<Box color="n11" flex="stretch">
 											{option.title}
 										</Box>
-										<Box color="n9" flex="stretch">
+										<Box
+											color="n9"
+											flex="stretch"
+											style={{
+												whiteSpace: 'nowrap',
+											}}
```

Comment:
There should be a `whiteSpace` prop available on `Box`. Could we use that instead of the `style` prop?